enatega-multivendor-app/src/screens/Checkout/Checkout.js

```
@@ -77,23 +62,14 @@ function Checkout(props) {
-    clearCart,
-    restaurant: cartRestaurant,
-    cart,
-    cartCount,
-    updateCart,
-    isPickup,
-    setIsPickup,
-    instructions
-  } = useContext(UserContext)
+  const { isLoggedIn, profile, clearCart, restaurant: cartRestaurant, cart, cartCount, updateCart, isPickup, setIsPickup, instructions } = useContext(UserContext)
+
+  console.log('rest===>', restaurant)
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)